rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -84,62 +74,87 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
     // check the vector size for bounding box. If its more than zero go for random bbox crop
     // else go to random crop
     unsigned int crop_width, crop_height, x1, y1, x1_diff, crop_width_diff;
-    if(_bbox_coord.size() != 0)
-    {
+    if(_bbox_coord.size() != 0) {
         // Random bbox crop returns normalized crop cordinates
         // hence bringing it back to absolute cordinates 
         x1 = std::lround(_bbox_coord[0] * original_image_width);
         y1 = std::lround(_bbox_coord[1] * original_image_height);
         crop_width = std::lround((_bbox_coord[2]) * original_image_width);
         crop_height = std::lround((_bbox_coord[3]) * original_image_height);
     }
-    else
-    {
-        std::vector<float> crop_mul_param =  decoder_config.get_crop_param();
-        auto is_valid_crop = [](uint h, uint w, uint height, uint width)
-        {
-            return (h < height && w < width); 
+    else {
+        double ASPECT_RATIO_RANGE[2] = {decoder_config.get_random_aspect_ratio()[0], decoder_config.get_random_aspect_ratio()[1]};
```

Comment:
Why is it declared in CAPS? Constants and #define only will be declared in caps.

Reply:
Okay, will make sure to follow this